persistence/persistence-spark/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>persistence</artifactId>
+        <groupId>org.mskcc.cbio</groupId>
+        <!-- project version is generated through git or can be passed as
+             PROJECT_VERSION env variable (see version.sh) -->
+        <version>0-unknown-version-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>persistence-spark</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.mskcc.cbio</groupId>
+            <artifactId>persistence-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
```

Comment:
Can we move this to top-level pom with other spring deps?
[https://github.com/cBioPortal/cbioportal/blob/master/pom.xml#L407](https://github.com/cBioPortal/cbioportal/blob/master/pom.xml#L407)